config/api/models/Page.php

```diff
@@ -45,12 +45,7 @@
             return $page->isSortable();
         },
         'next' => function (Page $page) {
-            return $page
-                ->nextAll()
-                ->filter('intendedTemplate', $page->intendedTemplate())
-                ->filter('status', $page->status())
-                ->filter('isReadable', true)
-                ->first();
+            return $page->panel()->next();
```

Comment:
Should we actually change these? I think I would rather deprecate them, they don't seem to make much sense for the API now that the Panel isn't using this anymore.

Reply:
I'm asking because I'm not entirely sure: Except of the Panel, isn't this used in API requests?

Reply:
Also, even if we deprecate it, I don't see any harm in changing it. Because it is one line and does the same job.

Reply:
I would deprecate them as well, but keep them for now with the added deprecate() call and remove the methods in 3.7.

Reply:
Is this fallback still correct after the last changes?

Reply:
Ahh, yes i've to revert this changes 👍